src/components/Layout/Layout.module.scss

```diff
@@ -3,3 +3,22 @@
   font-family: roboto;
   max-width: 100%;
 }
+
+.header {
+  text-align: center;
```

Comment:
Does `text-align: center` has any effect when you have `justify-content: center` & `align-items: center` ? Just wanted to know.

Reply:
Thanks for pointing it out @sumitd94 . 😄 It was indeed an extra line of code. Removed it. 👍

Reply:
Yep, Thank you :)